pkg/trace/config/config.go

```diff
@@ -459,6 +486,7 @@ func New() *AgentConfig {
 		AppSec: AppSecConfig{
 			Enabled:        true,
 			MaxPayloadSize: 5 * 1024 * 1024,
+			DDURL:          "",
```

Comment:
Can we remove this? The default is already empty string.

Reply:
Fixed in https://github.com/DataDog/datadog-agent/pull/11324/commits/9718f7d1bd2bde255789cd2697bf1ce59549b57d